src/store/index.ts

```diff
@@ -1,2 +1,3 @@
 export { audienceStore } from './audience';
 export { viewStore } from './pageview';
+export { audienceCache } from './audienceCache';
```

Comment:
We should update the naming, it is not clear the difference between a cache and a store.

Also should this be a new store/cache? (PS I know I said it should on our call)

Reply:
@k11kirky you did not answer my question here.

Reply:
@ydennisy Apologies, I think it is best to have it seperate for now, easier to test and read, but maybe a little more duplicate code. I have renamed the audience store in the lastest commits though